common/tags/src/main/java/org/apache/spark/annotation/InterfaceStability.java

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.annotation;
 
-import java.lang.annotation.Documented;
+import java.lang.annotation.*;
```

Comment:
No big deal, but I think we'd usually explicitly import 3 classes.

Reply:
Actually we need to import 5 classes, that's why my IDE turned it into * automatically...